test/c_api/test_ut_asserts.h

```diff
@@ -0,0 +1,74 @@
+/*
+ *
+ * Copyright (C) 2023-2024 Intel Corporation
+ *
+ * Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ */
+
+/*
+ The project uses GTEST framework for testing, which is not supported in C. Therefore, there have been introduced customised asserts for C API test. These asserts should not be used in other purposes than for testing C files 
```

Comment:
heh, this is a little too descriptive - comments should rather be short and concise.

also, please break lines. Each line should fit "in one screen", meaning, usually around 100-120 chars per line (we don't have strict limit for comments, I guess).

example:
```
/*
 * This project uses GTEST framework for testing, which is not supported in C.
 * These custom asserts can and should ONLY be used in C API tests!
 */
```

Reply:
Done:)